contentcuration/contentcuration/frontend/channelEdit/components/Clipboard/index.vue

```diff
@@ -248,9 +247,6 @@
           this.$analytics.trackClick('clipboard', 'Toggle clipboard');
         }
       },
-      channels() {
-        this.loadChannelColors();
-      },
```

Comment:
Removed since the clipboard action `addClipboardNodeFromListener` handles triggering `loadChannelColors` appropriately.